### PR TITLE
docs: gitlab pages for zh-cn & typo fix for github pages for zh-cn

### PR DESCRIPTION
**Pull request by SukkaW**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

#1108

### CODE REVIEW

File: source/zh-cn/docs/gitlab-pages.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 ---
 
-1. Create a new repository named <b>*username*.gitlab.io</b>, where username is your username on GitLab. If you have already uploaded to other repo, rename the repo instead.
-2. Enable Shared Runners via `Settings -> CI / CD -> Shared Runners`.
-3. Push the files of your Hexo folder to the repository. The `public/` folder is not (and should not be) uploaded by default, make sure the `.gitignore` file contains `public/` line. The folder structure should be roughly similar to [this repo](https://gitlab.com/pages/hexo).
-4. Add `.gitlab-ci.yml` file to your repo (alongside _config.yml & package.json) with the following content:
+在本教程中，我们将会使用 GitLab CI 将 Hexo 博客部署到 GitLab Pages 上。
+
+1. 新建一个 repository。如果你希望你的站点能通过 `<你的 GitLab 用户名>.gitlab.io` 域名访问，你的 repository 应该直接命名为 `<你的 GitLab 用户名>.gitlab.io`。
+2. 如果你使用的是 Self-Host 的 GitLab，你需要在 `Settings -> CI / CD -> Shared Runners` 启用 GitLab CI。
```
Comment (curbengh) [inline source/zh-cn/docs/gitlab-pages.md]: "Shared Runners" is available on Gitlab.com, not just self-host. It's enabled by default, so the step is just making sure.

File: source/zh-cn/docs/gitlab-pages.md [hunk 2]
```diff
@@ -25,19 +28,21 @@ pages:
...
-If you prefer to have a project page on GitLab:
+## Project page
 
-1. Go to `Settings -> General -> Advanced -> Change path`. Change the value to a name, so the website is available at <b>username.gitlab.io/*name*</b>. It can be any name, like *blog* or *hexo*.
-2. Edit **_config.yml**, change the `root:` value from `""` to `"name"`.
-3. Commit and push.
+如果你更希望你的站点部署在 `<你的 GitLab 用户名>.gitlab.io` 的子目录中，你的 repository 需要直接命名为子目录的名字，这样你的站点可以通过 `https://<你的 GitLab 用户名>.gitlab.io/<repository 的名字>` 访问。你需要检查你的 Hexo 配置文件，将 `url` 的值修改为 `https://<你的 GitLab 用户名>.gitlab.io/<repository 的名字>`、将 `root` 的值修改为 `/<repository 的名字>/`
 
+{% note info %}
+请注意，以上教程翻译自 Hexo 文档英文版，仅做参考；译者在翻译过程中发现很多语焉不详的内容。通过搜索引擎可以找到更多教程。
```
Comment (curbengh) [inline source/zh-cn/docs/gitlab-pages.md]: > 译者在翻译过程中发现很多语焉不详的内容
If there is any unclear instruction, you should've mention it in #1108.

Comment (SukkaW) [reply source/zh-cn/docs/gitlab-pages.md]: I have just copied the same note from `GitHub Pages` guide, as the guide provided by Hexo's documentation are  relatively unclear compared to the "hands-on Hexo teaching" tutorial articles written by Hexo users on their blogs.

It is not neccsary for Hexo document to cover all the use cases and give much more details. That is exactly what we do currently, providing a simple instruction.
But still, we need to inform Hexo users that there are many more use cases and details that is not included in Hexo documents.

Comment (curbengh) [reply source/zh-cn/docs/gitlab-pages.md]: Please remove that note from `GH Pages` guide as well, it shouldn't be there in the first place.

> we need to inform Hexo users that, there are many more use cases and details are not included in Hexo documents.

We don't need to, the instruction is meant to be generic for most use cases. Advanced users shouldn't rely on that. Besides, the note currently says "unclear instruction", not "doesn't cover all use cases".

Comment (SukkaW) [reply source/zh-cn/docs/gitlab-pages.md]: Removed.

File: source/zh-cn/docs/gitlab-pages.md (hunk 1 above)
Comment (NoahDragon) [reply source/zh-cn/docs/gitlab-pages.md]: Agree, "self-host" may make people think it refers to some other settings.

Comment (curbengh) [reply source/zh-cn/docs/gitlab-pages.md]: @SukkaW perhaps you could add a note at the end of this step saying this is usually enabled by default.

Comment (SukkaW) [reply source/zh-cn/docs/gitlab-pages.md]: Notes updated.

File: source/zh-cn/docs/gitlab-pages.md [hunk 3]
```diff
@@ -25,19 +27,20 @@ pages:
   only:
     - master
 ```
-5. *username*.gitlab.io should be up and running, once GitLab CI finishes the deployment job,
-6. (Optional) If you wish to inspect the generated site assets (html, css, js, etc), they can be found in the [job artifact](https://docs.gitlab.com/ee/user/project/pipelines/job_artifacts.html).
 
-### Project page
+5. GitLab CI 应该会自动开始运行，构建成功以后你应该可以在 `https://<你的 GitLab 用户名>.gitlab.io` 查看你的网站。
```
Comment (curbengh) [inline source/zh-cn/docs/gitlab-pages.md]: Step 4

**Pull request merged**